src/server/api/routes/routes.router.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+const express = require("express");
+const router = express.Router({
+  mergeParams: true
+});
+
+const routesController = require("../controllers/routes.controller");
+
+router.post("/", (req, res, next) => {
+  routesController
+    .createArouteWithWaypoints({
```

Comment:
There is a typo here. Should be `createRouteWithWaypoints`. Also, since we renamed routes to suggested routes, please call it `createSuggestedRouteWithWaypoints` and rename it in the controller as well.